src/sdltiles.cpp

```diff
@@ -3534,6 +3534,8 @@ void catacurses::init_interface()
     // initialize sound set
     load_soundset();
 
+    // Recreate font list;
+    save_font_list();
```

Comment:
`save_font_list` looks through many folders and opens all contained font files. This can take quite a while. Is it really necessary to do this on every start?

Reply:
> 
> 
> `save_font_list` looks through many folders and opens all contained font files. This can take quite a while. Is it really necessary to do this on every start?

Otherwise fonts are cached in `config/fontlist.txt` and no new fonts could be used unless this file is manually deleted and recreated.

Reply:
At this point, we might as well remove the file completely and just load the list of fonts here directly into memory. No point in storing it on disk when it's reloaded each time the program starts.

Reply:
On second thought, it is not really a necessary change to fix font loading issues, so I've reverted it.